managed/models/scheduled_tasks_helpers.go

```diff
@@ -39,7 +42,7 @@ func FindScheduledTaskByID(q *reform.Querier, id string) (*ScheduledTask, error)
 	case nil:
 		return res, nil
 	case reform.ErrNoRows:
-		return nil, status.Errorf(codes.NotFound, "ScheduledTask with ID %q not found.", id)
+		return nil, errors.Wrapf(ErrNotFound, "couldn't get scheduled task with ID %q", id)
```

Comment:
That was common approach for all such helpers. I guess we should refactor it everywhere then.

Reply:
Yeap, we should. API errors should be used only on API level otherwise we have a mess. But it should be done in separate PR I think.